### PR TITLE
Arriba wrapper

### PR DESCRIPTION
**Pull request by jafors**
Wrapper for the fusion detection tool arriba.

### CODE REVIEW

File: bio/arriba/test/Snakefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        fusions="fusions/{sample}.tsv",
+        # discarded gene fusions
+        discarded="fusions/{sample}.discarded.tsv" # optional
+    log:
+        "logs/arriba/{sample}.log"
+    params:
+        # path to reference genome
+        genome="genome.fasta",
+        # path to annotation gtf
+        annotation="annotation.gtf",
```
Comment (johanneskoester) [inline bio/arriba/test/Snakefile]: Above two can now be input files. I plan to modify all wrappers accordingly soon.

File: test.py [hunk 2]
```diff
@@ -106,6 +113,33 @@ def test_bedtools_slop():
...
+    )
+
+
+def test_bedtools_merge():
+    run("bio/bedtools/merge", ["snakemake", "A.merged.bed", "--use-conda", "-F"])
+
+
+def test_bedtools_slop():
+    run("bio/bedtools/slop", ["snakemake", "A.slop.bed", "--use-conda", "-F"])
+
```
Comment (johanneskoester) [inline test.py]: Where do all these tests come from? Have they been missing before?

File: bio/arriba/environment.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - arriba ==1.1.0
+  - star ==2.7.1a
```
Comment (johanneskoester) [inline bio/arriba/environment.yaml]: Why does it need star as a dependency? Star is not called in the wrapper.

Comment (jafors) [reply bio/arriba/environment.yaml]: Correct, I probably had it in there for testing it together with star.

File: bio/arriba/test/Snakefile (hunk 1 above)
Comment (jafors) [reply bio/arriba/test/Snakefile]: True, makes more sense to have those as inputs.

File: test.py (hunk 2 above)
Comment (jafors) [reply test.py]: They should have been included in my last PR actually. Maybe I messed up the commits.

**Pull request merged**